### PR TITLE
Data Sharing Chapter

### PR DESCRIPTION
**Pull request by cansavvy**
#### What changes are being implemented in this Pull Request?

What do you think of this? @carriewright11

### CODE REVIEW

**Comment by carriewright11** [comment]
looks good! 
Saw we got rid of this note: 
AVOCADO Somewhere want to talk about where you can access data... or share it...
https://transcriptomics.sequentiabiotech.com/
Repositories: GEO Gene Expression Omnibus(GEO), SRA(Sequencing Read Archive), ArrayExpress

Might want to still add [ArrayExpress](https://www.ebi.ac.uk/biostudies/ArrayExpress/studies), but maybe I missed seeing that somewhere.

**Comment by cansavvy** [comment]
> Saw we got rid of this note: 

I got rid of it once I felt it was covered, but feel free to add in or comment on what you believe is missing.

> Repositories: GEO Gene Expression Omnibus(GEO), SRA(Sequencing Read Archive), ArrayExpress

These are all linked to (or their parent database is).

> 
> Might want to still add [ArrayExpress](https://www.ebi.ac.uk/biostudies/ArrayExpress/studies), but maybe I missed seeing that somewhere.
> 

That's a part of EMBL which is linked

File: 04-Data_Sharing.Rmd [hunk 1]
```diff
@@ -1,24 +1,115 @@
...
+
+```{r, fig.align='center', echo = FALSE, fig.alt= "Ruby has uploaded her data to a repository and now its being used by many other researchers. Ruby says to her funders, represented as a bank, ‘The data you funded is getting so much mileage!’", out.width="100%"}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1SRokLaGAc2hiwJSN26FHE0ZEEhPr3KQdyMICic8kAcs/edit#slide=id.g117c57cc481_1_37")
+```
+
+## Data repositories
+
+The best way to share your data is by putting it somewhere that others can download it (and it can be kept private when necessary). There's many repositories out there that handle this for you.
+
+Below are some of the standard repositories for data you should consider. For a longer list of repositories, we also advise consulting this [Nature guidance on data repositories](https://www.nature.com/sdata/policies/repositories).
```
[the remark this answers is not in the document]
Comment (carriewright11) [reply 04-Data_Sharing.Rmd]: sounds good!

**Review by carriewright11 — APPROVED** [review]
Looks great! One thing I would like to cover is how to deal with data that requires HIPAA but also share your data responsibly if possible. I can make an issue about this and expanding on that resource you like

**Comment by cansavvy** [comment]
> Looks great! One thing I would like to cover is how to deal with data that requires HIPAA but also share your data responsibly if possible. I can make an issue about this and expanding on that resource you like

I'm working on covering that in the chapter before this, see #18, but it probably should be covered in more practical way in this chapter.

**Comment by carriewright11** [comment]
> > Looks great! One thing I would like to cover is how to deal with data that requires HIPAA but also share your data responsibly if possible. I can make an issue about this and expanding on that resource you like
> 
> I'm working on covering that in the chapter before this, see #18, but it probably should be covered in more practical way in this chapter.

Sounds good - I made an issue about it

**Pull request merged**